LICENSE.md

```diff
@@ -1,36 +1,25 @@
 ---
-layout: page
+layout: default
 title: LICENSE
 permalink: /license/
---- 
+---
```

Comment:
I removed the 80-character limit from `LICENSE.md`, since this is being rendered as part of the site and the line breaks were being faithfully carried into HTML.